Proposals/Proposal_02.md

```diff
@@ -0,0 +1,61 @@
+
+## Specification
+
+1. DAO Governance. Research and document DAO governance best practices and submit a detailed proposal recommending a governance plan for Volatility DAO by September 30, 2021 for comment with voting no later than October 7, 2021. Plan will cover:
+
+* Analysis of other DAO’s used as benchmarks
+* Proposed governance organization structure
+* Proposal process
+* Proposed voting methodology
+* Contributor and working group lead compensation, including compensation for ad hoc contributions
+* Systems required or enhancements to existing systems (e.g. use of POAPs within existing systems)
+* Preliminary DAO mission & objectives
+* Funding requirements
+
+2. Working Groups. Develop interim working group plans for Product, Marketing & Community working groups. Submit proposal for each including 90 day “startup” plan by September 30, 2021 for comment and voting no later than October 7, 2021. Each plan will cover:
+
+* Proposed working group mission
+* 90-day objectives
+* Team/talent requirements
+* Proposed interim leadership
+* Systems required
+* Proposed funding requirements
+* Founding Council Members
+
+The following participants will participate in the founding council:
+@aaadrew
+@Chuddymane
+@blasteasy /Kevin
+@Nakamomo
+@patb
+@secretwaves
+@sethmich
+@@wookie25
+Sandwedg
+@jonk
+
+In addition @Theta , @DarkKnight & @ ccashwell are invited to participate in this process as well.
+
+### Budget
+
+In the interests of ensuring a successful launch of the DAO and maximizing participation in this important process, a budget of no more than 142,00 VOL will be deployed against this effort.
+
+71,000 VOL will be allocated to participating members in the founding council and distributed on a pro-rata basis according to contribution effort. Founding council members will summarize their activities after four weeks in a Google Sheet and share with the founding council which will collectively agree how to distribute the budgeted VOL tokens. As a general guideline, those working on the council full-time (30+ hours per week) will be given greater (~2x) consideration than those contributing on a part-time (~10-20 hours) basis.
+
+An additional allocation of up to 71,000 VOL is requested for potential community or other third-party contributions in support of the founding council’s objectives. Approval for spending this allocation is subject to a majority of founding council members approving the amount with a “I agree” vote on the founding council discord channel.
+
+All VOL rewards will vest 100% after 180 days.
```

Comment:
Several people inquired why 6 months when most founding participants of an organization would normally have a much longer Vest length. It was brought up the minimum vest period for staking is currently 6 months and Vol founders / investors have a 2-year vest.  I would recommend we give the community optionality of 6, 12 or 24 months aligned. Maybe we consider vesting being released on a monthly increments so it's not all locked for the full period.